recipes/pymeos_cffi/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pymeos-cffi" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name | replace("-", "_") }}/{{ name | replace("-", "_") }}-{{ version }}.tar.gz
+  sha256: 08d0b50293f2af458188aa3a41af0d6ecd2d2db91130128ddf33ffb47343d57b
+
+build:
+  skip: true  # [not linux]
```

Comment:
What is happening on Windows and macOS? Can you get at lest macOS to work?

Reply:
When the PR was created MEOS only supported Linux, but now MacOS is supported too.
I'll work on making the python package available for MacOS too.
Thanks for pointing it out!!

Reply:
We managed to make the compilation work for macOS too, although in the way we found a couple issues that are fixed in the